### ISSUE TITLE
Error:(24, 13) Failed to resolve: com.echo:holographlibrary:1.0

### ISSUE THREAD

**Issue by hassanyousufx** [meleap/myo_AndoridEMG#11]
I am trying to import the project in Android Studio, I installed all the missing tools and plugins but I get an error in my build.gradle file saying

> Error:(24, 13) Failed to resolve: com.echo:holographlibrary:1.0

Preview:
[https://i.stack.imgur.com/rqk9O.png](url)

**Comment by cortinico** [comment]
Hey @hassanyf 
I guess this project is not yet maintained.

The error you're having is because the `holographlibrary` is not distributed via maven anymore. You probably need to download the library from https://github.com/Androguide/HoloGraphLibrary and provide an artifact inside the `libs` folder (or publish it on your mavenlocal).

Alternatively, I forked this repo and I rewrote the app from scratch providing also an Android library to interact with your Myo: https://github.com/cortinico/myonnaise
Maybe you can find it useful :)

**Comment by vikramharindranath** [comment]
Hello @cortinico ,

Thanks for the info. Were you able to extract EMG data from Myo with an Android app? If yes, can you help me how. 

Thanks in advance.

**Comment by cortinico** [comment]
> Thanks for the info. Were you able to extract EMG data from Myo with an Android app? If yes, can you help me how.

Yes it's everything described on my repo README https://github.com/cortinico/myonnaise
